src/frontend-pwa/src/views/Report/report.styles.ts

```diff
@@ -0,0 +1,93 @@
+import styled from '@emotion/styled';
+import {
+  TextField,
+  TextFieldProps,
+  Select,
+  SelectProps,
+  MenuItem,
+  MenuItemProps,
+} from '@mui/material';
+import BCSans from '/BCSans-Regular.woff';
+import typography from '../../typography';
+
+export const ReportWrapperDiv = styled.div`
+    width: 100%;
+    height: 100%;
+    padding-top: 1em;
+    margin: auto; 
+    width: 30em; 
+    position: relative;
+    // display: inline-block;
+    overflow: auto;
+`;
+
+export const ReportOuterDiv = styled.div`
+    width: 100%;
+    max-height: 100%;
+    min-width: 0;
+    min-height: 0;
+    display: flex;
+    flex-direction: column;
+    // align-items: center;
```

Comment:
I would just remove these instead of commenting them out. It's likely that this will never be removed after being commented out and it will just be useless code that sits there.